src/web/components/Sidebar/Sidebar.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Sidebar from './';
+
+describe('Sidebar', () => {
+  let wrapper;
+  let children;
+  let content;
+
+  beforeEach(() => {
+    children = <div>children</div>;
+    content = <div>content</div>;
+    wrapper = shallow(<Sidebar content={content}>{children}</Sidebar>);
+  });
+
+  it('should render children component', () => {
+    expect(wrapper.contains(children));
+  });
+
+  it('should render sidebar content', () => {
+    expect(wrapper.contains(children));
+  });
+
+  it('should render a div with #efefef backgorund color by default', () => {
+    const color = wrapper
+      .find('div')
+      .children()
+      .first()
+      .prop('style').backgroundColor;
+
+    expect(color).toBe('#efefef');
+  });
+
+  it('should render a div with #eeeeee backgorund color', () => {
+    wrapper.setProps({ color: '#eeeeee' });
+    const color = wrapper
+      .find('div')
+      .children()
+      .first()
+      .prop('style').backgroundColor;
+
+    expect(color).toBe('#eeeeee');
+  });
+
+  it('should render a div with 160px width by default', () => {
+    const color = wrapper
+      .find('div')
+      .children()
+      .first()
+      .prop('style').width;
+
+    expect(color).toBe('160px');
```

Comment:
Expected a color to equal some pixels is weird 😝

Reply:
Done :smile: